content/docs/ops/design/secrets.md

```diff
@@ -158,7 +158,75 @@ Concourse, and CredHub.
 
 ## Bootstrapping CredHub
 
-> To Be Decided
+Deploying anything before there is a CredHub available on the BOSH director
+you're deploying to requires you to deploy CredHub manually using established
+`operations` and `variables` files that leverage BOSH interpolate.
+
+### Generating import file
+
+To ease the import of secrets into a fresh deployment of CredHub, take the
+`common/secrets.yml` file run it through Pivotal's `vars-to-credhub` tool.
+
+Get the name from the BOSH director using `bosh environment`.
+
+```sh
+$ bosh environment --tty | grep Name | awk '{ print $2 }'
+```
+Get the name of the deployments available in BOSH by using `bosh deployments`.
+
+```sh
+bosh deployments --column=Name
+```
+
```

Comment:
The steps here need to include getting the `common-secrets` for the deployment before running `vars-to-credhub`. Also, this would probably be a good section to link to the repo and explain how to install it. or have just have the README there explain it.